barretenberg/cpp/src/barretenberg/relations/generated/avm/avm_alu.hpp

```diff
@@ -6,1215 +6,1165 @@
 
 namespace bb::Avm_vm {
 
```

Comment:
i think the formatter has messed up here

Reply:
weird, i thought it auto-formatted as a hook. I'll re-run the format script